react/src/components/relations-app/remote/loadRelations.js

```diff
@@ -0,0 +1,15 @@
+const relations = new Array(100).fill(undefined).map((_, index) => {
```

Comment:
Because you've done it at several places, a small info: `new Array(n)` fills an Array per default with `undefined` and there is no need to call `fill()`.

And another small info for the unlikely case that you don't know it: you can return a newly created object in an arrow function without return by using `()`. That means, that you could have written:
`.map((_, index) => ({ title: `Relation${index}` }));`